public/index.html

```diff
@@ -2,12 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <!-- zendesk requires a referrer to be set for the widget to work
-         * no-referrer is used in the private environment without zendesk, the most secure option
-         * origin is the next most restrictive option after no-referrer, used in the app environment,
-           which exposes app.shapeshift.com as the referrer to the widget
-    -->
-    <meta name="referrer" content="%REACT_APP_REFERRER%">
+    <meta name="referrer" content="no-referrer">
```

Comment:
putting this back to what it was previously, the most strict version, `no-referrer`